src/errors/runtime/templates.js

```diff
@@ -15,6 +15,7 @@ export default {
     [RUNTIME_ERRORS.cannotRunLiveModeRunnerMultipleTimes]: 'Cannot run a live mode runner multiple times.',
     [RUNTIME_ERRORS.browserDisconnected]:                  'The {userAgent} browser disconnected. This problem may appear when a browser hangs or is closed, or due to network issues.',
     [RUNTIME_ERRORS.cannotRunAgainstDisconnectedBrowsers]: 'The following browsers disconnected: {userAgents}. Tests will not be run.',
+    [RUNTIME_ERRORS.testRunRequestInDisconnectedBrowser]:  'The requested "{browser}" browser was disconnected during the test execution',
```

Comment:
```suggestion
    [RUNTIME_ERRORS.testRunRequestInDisconnectedBrowser]:  'The requested "{browser}" browser has disconnected during the test execution',
```

Assuming that the browser disconnected at some point during the test.

Reply:
Fixed, please review